3-node-cluster/deploy.py

```diff
@@ -105,6 +106,58 @@ def CHECK_VAULT_PASSWORD_FILE(FILE_PATH):
     # All conditions passed
     return True
 
+# Function to input only valid IP addresses
+def GET_VALID_IP(PROMPT, SUBNET, EXISTING_IPS=[]):
+    INVALID_INPUTS = 0
+
+    while True:
+        IP = input(PROMPT)
+        if not IP:
+            print("Empty input. Please enter a valid IP address.")
+            INVALID_INPUTS += 1
+            if INVALID_INPUTS >= 3:
+                print("Too many invalid inputs. Exiting pg_cirrus.")
+                exit()
+            continue
+        try:
+            IP_OBJECT = ipaddress.ip_address(IP)
+            if IP_OBJECT not in ipaddress.ip_network(SUBNET):
+                raise ValueError("Invalid IP address or not within the cluster subnet.")
+            if IP in [SERVER['IP'] for SERVER in EXISTING_IPS]:
+                raise ValueError("IP address is already added as a Primary or Standby node.")
+            if not subprocess.call(['ping', '-c', '1', str(IP)]) == 0:
```

Comment:
Please don't display output for ping command on the screen as it might effect user experience of pg_cirrus execution
Instead what we can do

Suggestion) We should throw its output to /dev/null so it shouldn't be displayed to the screen. We can even write a function for ping and call that function here

Reply:
Output of ping command not being displayed on screen now. Output is now being hidden using ```stdout=subprocess.DEVNULL```. Kindly review and merge.